src/WebJobs.Extensions.DurableTask/StatusResponsePayload.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Runtime.Serialization;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Response for Orchestration Status Query.
+    /// </summary>
+    [DataContract]
+    public class StatusResponsePayload
```

Comment:
Can this be internal? I don't see a reason why developers would need to reference this directly.

Reply:
@cgillum  Good point. I will modify it. Thank you!